codex/Main.hs

```diff
@@ -75,7 +76,10 @@ writeCacheHash cx = writeFile $ hashFile cx
 
 update :: Bool -> Codex -> Builder -> IO ()
 update force cx bldr = displayConsoleRegions $ do
-  (mpid, dependencies, workspaceProjects') <- resolveCurrentProjectDependencies bldr $ hackagePath cx </> "00-index.tar"
+  (mpid, dependencies, workspaceProjects') <- catchJust
```

Comment:
This solution is wrong. Please use the proper way to discover the index file. https://github.com/aloiscochard/codex/pull/88/commits/3e6b01de283a8e5ac2e5d153cb2fd88b574b8b03

See https://github.com/aloiscochard/codex/pull/88